### ISSUE TITLE
Panic when dropping db / snapshotting

### ISSUE THREAD

**Issue by e-dard** [influxdata/influxdb#8721]
When dropping a large database I encountered this:

```
Aug 18 13:52:05 ip-10-0-136-2 influxd[2552]: panic: runtime error: invalid memory address or nil pointer dereference
Aug 18 13:52:05 ip-10-0-136-2 influxd[2552]: [signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x85437a]
Aug 18 13:52:05 ip-10-0-136-2 influxd[2552]: goroutine 262319 [running]:
Aug 18 13:52:05 ip-10-0-136-2 influxd[2552]: github.com/influxdata/influxdb/tsdb.(*Shard).CreateSnapshot(0xc99efd2380, 0x0, 0x0, 0x0, 0x0)
Aug 18 13:52:05 ip-10-0-136-2 influxd[2552]:         /go/src/github.com/influxdata/influxdb/tsdb/shard.go:972 +0x9a
Aug 18 13:52:05 ip-10-0-136-2 influxd[2552]: github.com/influxdata/influxdb/tsdb.(*Store).CreateShardSnapshot(0xc4202382c0, 0xd, 0x32, 0x0, 0x0, 0x0)
Aug 18 13:52:05 ip-10-0-136-2 influxd[2552]:  
```

Not sure if this will possibly be fixed via https://github.com/influxdata/influxdb/pull/8714

**Comment by e-dard** [comment]
I should add this was when running `1.3.3`

**Comment by e-dard** [comment]
Fixed via #8757
